lib/template-mixins.js

```diff
@@ -139,12 +139,17 @@ module.exports = function (options, deprecated) {
         var getTemplate = function getTemplate(child) {
             var re = /^partials\/(.+)/i;
             var match = child.match(re);
+            res.locals.partials = res.locals.partials || {};
             if (match) {
-                res.locals.partials = res.locals.partials || {};
                 return fs.readFileSync(res.locals.partials['partials-' + match[1]] + '.' + viewEngine).toString();
             } else if (child === 'html' || res.locals[child]) {
                 var panelPath = path.join(viewsDirectory, PANELMIXIN + '.' + viewEngine);
-                return fs.readFileSync(panelPath).toString();
+                var localsPanelPath = path.join(res.locals.partials['partials-mixins-panel'] + '.' + viewEngine);
+                if (res.locals.partials['partials-mixins-panel']) {
```

Comment:
I don't think this is legit. We check for a path on `res.locals` and then if it exists, use something else. If you look up to line 144 you'll see that the `res.locals.partials` is the path that you should read.